widgets/projection/powder/powder.ui

```diff
@@ -33,33 +33,7 @@
          </widget>
         </item>
         <item row="0" column="1">
-         <widget class="QComboBox" name="cmbPowderU1">
-          <item>
-           <property name="text">
-            <string>Energy</string>
-           </property>
-          </item>
-          <item>
-           <property name="text">
-            <string>|Q|</string>
-           </property>
-          </item>
-          <item>
-           <property name="text">
-            <string>2Theta</string>
-           </property>
-          </item>
-          <item>
-           <property name="text">
-            <string>Det Group Number</string>
-           </property>
-          </item>
-          <item>
-           <property name="text">
-            <string>Time Channel</string>
-           </property>
-          </item>
-         </widget>
```

Comment:
The .ui file was found to contain a list of initial entries. On contruction the presenter populates the drop downs with values stored in the presenter, effectively making this *Dead Code*